### ISSUE TITLE
Import multi-column CSV lines with last column being empty

### ISSUE THREAD

**Issue by kafeijao** [VictoriaMetrics/VictoriaMetrics#4048]
### Describe the bug

I'm having issues when importing a multi-column CSV lines with null-valued fields. It works as it should except when my CSV last column is empty. Which results in:

`error when parsing csv line "1677633061449998000,1,": missing columns in the csv line "1677633061449998000,1,"; got 2 columns; want at least 3 columns; skipping this line`

For this test I used:
**Endpoint:** `/api/v1/import/csv`
**Format:** `1:time:unix_ns,2:metric:metric_1,3:metric:metric_2`
**Payload:**
```
1677632461449998000,1,1
1677633061449998000,1,
1677633661449998000,,1
```

Related:
https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3540
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3565

### To Reproduce

Attempt to ingest data using a post request to `/api/v1/import/csv` endpoint

Use query arg **format**: `1:time:unix_ns,2:metric:metric_1,3:metric:metric_2`
Use the payload:
```
1677632461449998000,1,1
1677633061449998000,1,
1677633661449998000,,1
```

This will ingest the first and third row of the CSV, but the second one will be skipped with the error:
`error when parsing csv line "1677633061449998000,1,": missing columns in the csv line "1677633061449998000,1,"; got 2 columns; want at least 3 columns; skipping this line`

The second row as we can see in the payload has 3 columns, except the last one is empty. But this throws an error preventing to ingest this row.

### Version

`victoria-metrics-20230326-001737-tags-v1.87.4-0-g87769b36d`

### Additional information

As a temporary bypass for this issue it seems I can append a `,` at the end of rows that end with an empty value. So this would work (to ingest 3 columns):
```
1677633061449998000,1,,
```

**Comment by valyala** [comment]
@kafeijao , thanks for reporting the bug! It must be fixed in the commit 160453b86c90178ec7dab66ba23d5102f48f17ff . This commit will be included in the next release. In the mean time you can build VictoriaMetrics from this commit according to [these instructions](https://docs.victoriametrics.com/#how-to-build-from-sources) and verify whether this fixes the issue with improper parsing of last empty column in CSV.

**Comment by valyala** [comment]
The bugfix has been included in [v1.91.0 release](https://docs.victoriametrics.com/CHANGELOG.html) and also has been backported to [v1.87.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.6).

Closing the issue then.
